.github/actions/is-release-in-progress-v1/src/run.ts

```diff
@@ -0,0 +1,24 @@
+import type { Core, Github, PullRequest } from './types'
+import isReleaseInProgress from './is-release-in-progress'
+
+export default async function run(core: Core, github: Github) {
+  try {
+    const githubToken = core.getInput('githubToken', { required: true })
+
+    // Fetch open pull requests
+    const octokit = github.getOctokit(githubToken)
+    const { data: pullRequests } = await octokit.rest.pulls.list({
+      ...github.context.repo,
+      state: 'open'
+    })
+
+    // Set output
+    core.setOutput(
+      'isReleaseInProgress',
+      isReleaseInProgress(pullRequests as unknown as PullRequest[])
+    )
+    core.info('Set isReleaseInProgress output')
```

Comment:
We should output the result of `isReleaseInProgress` to make debugging easier

```suggestion
    const releaseInProgress = isReleaseInProgress(pullRequests as unknown as PullRequest[])
    
    // Set output
    core.setOutput('isReleaseInProgress', releaseInProgress)
    core.info('Set isReleaseInProgress output: ', releaseInProgress)
```